client/state/data-layer/wpcom/jetpack/settings/index.js

```diff
@@ -127,13 +131,19 @@ export const handleSaveSuccess = (
 	{ data: { code, message, ...updatedSettings } } // eslint-disable-line no-unused-vars
 ) => dispatch( saveJetpackSettingsSuccess( siteId, updatedSettings ) );
 
-export const announceSaveFailure = ( { dispatch }, { siteId } ) =>
+export const announceSaveFailure = (
```

Comment:
This is not named so well now that it does other stuff.

Reply:
Good point. Hmm, naming. `announceSaveFailureAndReset` maybe?